### PR TITLE
fix foreground hangup on android Pie

### PR DESCRIPTION
**Pull request by Barmil**
Run a foreground service while handling a call with hangup notification attached to it

### CODE REVIEW

File: android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java [hunk 1]
```diff
@@ -392,8 +416,8 @@ private void handleIncomingCallIntent(Intent intent) {
                 }
                 SoundPoolManager.getInstance(getReactApplicationContext()).playRinging();
 
-                if (getReactApplicationContext().getCurrentActivity() != null) {
-                    Window window = getReactApplicationContext().getCurrentActivity().getWindow();
+                if (getCurrentActivity() != null) {
```
Comment (noamSoluto) [inline android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java]: why did you remove the `getReactApplicationContext()`?

Comment (Barmil) [reply android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java]: errors on on android studio, no need fo it

File: android/src/main/java/com/hoxfon/react/RNTwilioVoice/CallForegroundService.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        {
+            Log.e(TAG, "error while starting call foreground service:\n" + ex.toString());
+
+        }
+        return super.onStartCommand(intent, flags, startId);
+    }
+
+    @Override
+    public void onDestroy() {
+        Log.d("RNTwilioVoice", "onDestroy");
```
[the remark this answers is not in the document]
Comment (Barmil) [reply android/src/main/java/com/hoxfon/react/RNTwilioVoice/CallForegroundService.java]: yeah, you're right
